src/compiler/checker.ts

```diff
@@ -10449,7 +10449,10 @@ namespace ts {
                     case SyntaxKind.FunctionExpression:
                     case SyntaxKind.FunctionDeclaration:
                     case SyntaxKind.ArrowFunction:
-                    case SyntaxKind.ObjectLiteralExpression: return false;
+                    case SyntaxKind.ClassExpression:
+                    case SyntaxKind.ClassDeclaration:
+                    case SyntaxKind.GetAccessor:
+                    case SyntaxKind.SetAccessor: return false;

```

Comment:
Okay, but just so you know, it is not consistent with `yield` and `await` if we allow it in a computed property of a class.